pset-functions.js

```diff
@@ -136,6 +177,43 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function scrabbleScore(word){
+const letters = word.split("")
+const onePoint = ["a","e","i","o","u","l", "n", "r", "s", "t"] 
+const twoPoints = ["d","g"] 
+const threePoints = ["b","c","m","p"] 
+const fourPoints = ["f","h","v","w","y"] 
+const fivePoints = ["k"] 
+const eightPoints = ["j","x"] 
+const tenPoints = ["q","z"] 
+
+let finalScore = 0
+
+letters.forEach((letter) => {
+  if(onePoint.includes(letter)){
```

Comment:
@Tambini nice use of includes to determine if the letter is found in the different array groups